static/config/status-chain-genesis.json

```diff
@@ -0,0 +1,31 @@
+{
+  "alloc": {
+     "AdAf150b905Cf5E6A778E553E15A139B6618BbB7": {
+      "balance": "0x10F0CF064DD59200000"
+    },
+    "65C01586aa0Ce152835c788aCe665e91Ab3527b8": {
+      "balance": "0x1B1AE4D6E2EF500000"
+    }
+  },
+  "coinbase": "0x0000000000000000000000000000000000000000",
```

Comment:
Let's revise the genesis block. I'd like it to be easier to grasp and we can do that provided that most default values should be fine.
For example, `coinbase` will be `0x0000000000000000000000000000000000000000`, specifying it here doesn't give any valuable information. Let's remove it.